worker/wait_group.go

```diff
@@ -0,0 +1,30 @@
+package worker
+
+import "sync"
+
+type waitGroupWithCount struct {
+	sync.WaitGroup
+
+	countMutex sync.Mutex
+	count      int
+}
+
+func (bwg *waitGroupWithCount) Increment() {
```

Comment:
I'm confused why the ptr receiver is named `bwg`; What's the "b", b?

Reply:
No idea. I copied this from its previous location (`worker/beacon`). I can rename it.